product-service/src/services/productService/index.ts

```diff
@@ -0,0 +1,10 @@
+import products from '../../mocks/products.json';
+import { Product } from '../../types'
+
+export async function getAll(): Promise<Product[]> {
+  return products;
```

Comment:
technically returning array or finding element in the array i not async operation, will be better to simulate with returning Promise

Reply:
> technically returning array or finding element in the array i not async operation, will be better to simulate with returning Promise

Thanks, I've missed that.